APITest/APITest/Models/EmployeeModel.cs

```diff
@@ -1,6 +1,11 @@
 ﻿namespace APITest.Models
 {
-    public class EmployeeModel
+    public class EmployeeModel : BaseModel
     {
+        public int? id { get; set; }
```

Comment:
Are you sure that you need nullable here?

Reply:
Yes, it is necessary for the Post method. Because of the server itself generate id randomly. And my id not need